types/react/test/tsx.tsx

```diff
@@ -156,6 +156,7 @@ class ComponentWithLargeState extends React.Component<{}, Record<'a'|'b'|'c', st
         return { a: 'a' };
     }
 }
+const AssignedComponentWithLargeState: React.ComponentClass<{}> = ComponentWithLargeState;
```

Comment:
I added one anyway; though this one wouldn't have failed even without the change because of the `tsconfig.json`.